charts/datadog-operator/templates/deployment.yaml

```
@@ -3,6 +3,8 @@ kind: Deployment
 metadata:
   name: {{ include "datadog-operator.fullname" . }}
   namespace: {{ .Release.Namespace }}
+  annotations:
+{{ include "datadog-operator.deployment.annotations" . | indent 4 }}
```

for `labels` we have a helper function that compute the labels from what is provided by the user with `.Values.deployment.labels` and default labels that we always want to add.

but for annotations we don't have this kind of feature so the this line should be replace by 

```suggestion
{{ .Values.deployment.annotations" | indent 4 }}
```